open-metadata-test/open-metadata-fvt/access-services-fvt/stewardship-action-fvt/build.gradle

```diff
@@ -59,7 +59,7 @@ task startServer(dependsOn: ['cleanData',':open-metadata-distribution:open-metad
     // Start some external service
     executable = System.properties.'java.home' + '/bin/java'
     workingDir = "${buildDir}"
-    args = [ "-Dserver.port=10445",
+    args = [ "-Dserver.port=10453",
```

Comment:
This change introduces a potential port clash when running parallel builds in gradle (possibly maven)
The ports must be unique across all FVTs otherwise build ordering could lead to random failures in future